addons/website_event/static/src/snippets/s_events/000.js

```diff
@@ -0,0 +1,35 @@
+odoo.define('website_event.s_events', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
```

Comment:
let ? just to be using const / let new naming ?

Reply:
`const` :wink: